core/src/main/resources/org/apache/spark/ui/static/stagepage.js

```diff
@@ -0,0 +1,926 @@
+                                            return str;
+                                            break;
+
+                                        case 'Shuffle Read Size / Records':
+                                            var str1arr = extractDataFromArrayString(JSON.stringify(row.data.readBytes));
+                                            var str2arr = extractDataFromArrayString(JSON.stringify(row.data.readRecords));
+                                            var str = formatBytes(str1arr[4], type) + " / " + str2arr[4];
+                                            return str;
+                                            break;
+
+                                        case 'Shuffle Read Blocked Time':
+                                            var str1arr = extractDataFromArrayString(JSON.stringify(row.data.fetchWaitTime));
+                                            var str = formatDuration(str1arr[4]);
+                                            return str;
+                                            break;
+
+                                        case 'Shuffle Remote Reads':
+                                            var str1arr = extractDataFromArrayString(JSON.stringify(row.data.remoteBytesRead));
+                                            var str = formatBytes(str1arr[4], type);
+                                            return str;
+                                            break;
+
+                                        case 'Shuffle Write Size / Records':
+                                            var str1arr = extractDataFromArrayString(JSON.stringify(row.data.writeBytes));
+                                            var str2arr = extractDataFromArrayString(JSON.stringify(row.data.writeRecords));
+                                            var str = formatBytes(str1arr[4], type) + " / " + str2arr[4];
+                                            return str;
+                                            break;
+
+                                        default:
+                                            return (row.metric == 'Peak Execution Memory' || row.metric == 'Shuffle spill (memory)'
+                                                    || row.metric == 'Shuffle spill (disk)') ? formatBytes(row.data[4], type) : (formatDuration(row.data[4]));
+
+                                    }
+                                }
+                            }
+                        ],
+                        "columnDefs": [
+                            { "type": "file-size", "targets": 1 },
+                            { "type": "file-size", "targets": 2 },
+                            { "type": "file-size", "targets": 3 },
+                            { "type": "file-size", "targets": 4 },
+                            { "type": "file-size", "targets": 5 }
+                        ],
+                        "paging": false,
+                        "searching": false,
```

Comment:
what is searching for the aggregated metrics table based on?  If you search for an integer like 32 (which would match an id) it finds other things but also shows rows with no 32 in it.  I also assume the search is based on the unformatted numbers because I tried ot search for 1.2 GB and 1.2 which should have matched on the Input Size but it showed nothing

Reply:
Yep we can discuss that.